ql/src/test/org/apache/hadoop/hive/ql/exec/vector/expressions/TestVectorTypeCasts.java

```diff
@@ -24,13 +24,15 @@
 
 import java.math.BigDecimal;
 import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
 import java.util.Random;
 import java.util.concurrent.TimeUnit;
 
 import junit.framework.Assert;
 
 import org.apache.hadoop.hive.common.type.DataTypePhysicalVariation;
 import org.apache.hadoop.hive.common.type.HiveDecimal;
+import org.apache.hadoop.hive.ql.util.DateTimeMath;
```

Comment:
testCastDateToString() is missing in branch-3 but in master. Let's cherry pick that test.

Reply:
Taken care of in https://github.com/apache/hive/pull/4674/files#diff-338c30d446a4b08aad91febdd0734bad0bca709efe70b9396e878a0215cd6155